### PR TITLE
[SP-6256] Moving LogzIO step to new standard

### PR DESCRIPTION
**Pull request by igorkheif**

### CODE REVIEW

File: steps/logzio/base/base.go [hunk 1]
```diff
@@ -1,6 +1,92 @@
...
+	err = json.Unmarshal(rawRespBody, &respBody)
+	if err != nil {
+		v := fmt.Sprintf("error: %q, response body: %q", err.Error(), string(rawRespBody))
+		return nil, nil, status.New(codes.Internal, "failed unmarshaling response body", v)
+	}
+
+	return rawRespBody, respBody, status.OK
+}
+
+func GetStatus(httpCode int) status.Status {
```
Comment (eldadru) [inline steps/logzio/base/base.go:73]: Sorry for jumping into this PR uninvited - this looks like a re-useable code - WDYT?

Comment (igorkheif) [reply steps/logzio/base/base.go:73]: It is, I am planning to move it later on somewhere more `common` (in fact the same code is used in different steps atm).

File: steps/logzio/base/base.go [hunk 2]
```diff
@@ -1,6 +1,92 @@
...
+	case httpCode/100 == 2:
+		retStatus = status.OK
+	case httpCode == http.StatusNotFound:
+		retStatus = status.New(codes.NotFound, "resource not found", "")
+	case httpCode == http.StatusUnauthorized:
+		retStatus = status.New(codes.Unauthenticated, "authentication failed, token parameters", "")
+	case httpCode/100 == 4:
+		retStatus = status.New(codes.InvalidArgument, "encountered a client side error, please check all parameters and try again", "")
+	case httpCode/100 == 5:
+		retStatus = status.New(codes.Internal, "coralogix has encountered an internal error", "")
```
Comment (hagaishapira) [inline steps/logzio/base/base.go:86]: This is logz. I think this is a good sign this code is similar :)

**Review by hagaishapira — CHANGES_REQUESTED** [review]
Fix type, other than that LGTM

File: steps/logzio/base/base.go (hunk 2 above)
Comment (igorkheif) [reply steps/logzio/base/base.go]: True. Fixed!

**Review by hagaishapira — APPROVED** [review]
LGTM

**Review by oriser — APPROVED** [review]
Agree with @eldadru comment, we should move it to the SDK, other than that - LGTM

**Pull request merged**